ibis/expr/visualize.py

```diff
@@ -41,37 +41,52 @@ def get_type(node):
         ]
         schema = ibis.schema(pairs)
 
-    return (
-        ''.join(
-            f'<BR ALIGN="LEFT" /> <I>{escape(name)}</I>: {escape(str(type))}'
-            for name, type in zip(schema.names, schema.types)
-        )
-        + '<BR ALIGN="LEFT" />'
+    return '<BR ALIGN="LEFT" />' + '<BR ALIGN="LEFT" />'.join(
+        f'<I>{escape(name)}</I>: {escape(str(type))}'
+        for name, type in zip(schema.names, schema.types)
     )
 
 
 def get_label(node):
     typename = get_type(node)  # Already an escaped string
+    # If the typename is of the type array<[type]>, e.g. array<int8>,
+    # graphviz will try to render <int8> as an HTML tag and fail
+    if typename.startswith("array<") and typename.endswith(">"):
+        typename = typename.replace("<", "[")
+        typename = typename.replace(">", "]")
```

Comment:
Would `urllib.parse.quote_plus(typename)` also work?

Reply:
Nevermind, there's a different way to do this!

Reply:
Went ahead and did it in the interest of landing the PR!

Reply:
Thank you!